miso-web/src/main/webapp/scripts/list.js

```diff
@@ -298,6 +298,29 @@ ListUtils = (function($) {
           updateSelectedLabel(state);
         }
       });
+      staticActions.push({
+        "name": "🗹",
```

Comment:
I don't know what this is supposed to be, but on my browser it shows up as one of those unknown character blocks?
![image](https://user-images.githubusercontent.com/17747491/64171946-e0118d00-ce21-11e9-96ec-f14b668860cb.png)

Reply:
I � Unicode